src/bindings/python/nixl_bindings.cpp

```
@@ -691,6 +782,135 @@ PYBIND11_MODULE(_bindings, m) {
+                }
+                for (auto &t : threads) t.join();
+            },
+            py::arg("reqhs"),
+            py::arg("num_threads") = 0,
+            py::call_guard<py::gil_scoped_release>())
+        .def(
+            "parallelStorageTransfer",
+            [](nixlAgent &agent,
+               std::vector<std::vector<std::tuple<uint64_t, uint64_t, int>>> file_desc_lists,
+               const std::string &agent_name,
+               std::vector<uintptr_t> backends,
```

Cppcheck (2.20.0)

[performance] Function parameter 'backends' should be passed by const reference.

(passedByValue)